server/plugin/api.go

```diff
@@ -979,6 +982,81 @@ func (p *Plugin) getPrByNumber(w http.ResponseWriter, r *http.Request, userID st
 	p.writeJSON(w, result)
 }
 
+func (p *Plugin) getLabels(w http.ResponseWriter, r *http.Request, userID string) {
+	info, apiErr := p.getGitHubUserInfo(userID)
+	if apiErr != nil {
+		p.writeAPIError(w, apiErr)
+		return
+	}
+
+	owner, repo, err := parseRepo(r.URL.Query().Get("repo"))
+	if err != nil {
+		p.writeAPIError(w, &APIErrorResponse{Message: err.Error(), StatusCode: http.StatusBadRequest})
+		return
+	}
+
+	githubClient := p.githubConnect(*info.Token)
+	result, _, err := githubClient.Issues.ListLabels(context.Background(), owner, repo, &github.ListOptions{})
```

Comment:
All list API methods from the `github` package must use pagination. Otherwise only the first 30 options will be show. See https://pkg.go.dev/github.com/google/go-github/v31/github?tab=doc#hdr-Pagination for more information and sample code.